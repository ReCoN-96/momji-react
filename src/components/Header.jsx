import {
  Menu,
  Image,
  Box,
  Heading,
} from '@chakra-ui/react';
import { Link } from 'react-router-dom';

const Header = () => (
  <Menu>
    <Box boxSize="100%" p={10} display="flex" justifyContent="center" alignItems="center">
      <Link to="/">
        <Image src="https://www.momji.fr/sites/default/files/logo/SA_to_Momji_80px_1.png" alt="Momji" />
      </Link>
      <Heading ml={10}>Test Technique Momji</Heading>
    </Box>
  </Menu>
);

export default Header;
