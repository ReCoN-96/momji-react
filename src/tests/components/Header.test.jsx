/* eslint-disable no-undef */
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Header from '../../components/Header';

test('should render Header correctly', () => {
  const wrapper = renderer.create(<Router><Header /></Router>).toJSON();
  expect(wrapper).toMatchSnapshot();
});
