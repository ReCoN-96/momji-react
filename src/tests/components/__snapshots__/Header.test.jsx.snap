// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<Menu>
  <Box
    alignItems="center"
    boxSize="100%"
    display="flex"
    justifyContent="center"
    p={10}
  >
    <Link
      to="/"
    >
      <Image
        alt="Momji"
        src="https://www.momji.fr/sites/default/files/logo/SA_to_Momji_80px_1.png"
      />
    </Link>
    <Heading
      ml={10}
    >
      Test Technique Momji
    </Heading>
  </Box>
</Menu>
`;
